Sources/UberAuth/AuthManager.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Copyright © Uber Technologies, Inc. All rights reserved.
+//
+
+
+import Foundation
+import UIKit
+
+public typealias AuthCompletion = (Result<Client, UberAuthError>) -> ()
+
+/// @mockable
+protocol AuthManaging {
+    
+    func login(context: AuthContext, completion: @escaping AuthCompletion)
+    
+    func handle(_ url: URL) -> Bool
+}
+
+final class AuthManager: AuthManaging {
+    
+    private var currentContext: AuthContext?
+    
+    init() {}
```

Comment:
NIT: since this is a non-plublic class, can we remove this?

Reply:
Fixed